.github/workflows/golangci-lint.yml

```diff
@@ -53,8 +53,5 @@ jobs:
           # into fixing all linting issues in the whole file instead
           args: --timeout=30m --whole-files
 
-          # Optional: if set to true then the action will use pre-installed Go.
-          skip-go-installation: true
```

Comment:
This was deprecated in 

```
v4.0.0+ requires an explicit actions/setup-go installation step before using this action: uses: actions/setup-go@v5. The skip-go-installation option has been removed.
```

But we are running version `v1.55.2`, so I am not sure removing this achieves what you want.
Why we wan to remove this line?

Reply:
I will revert this change now.